packages/browser-agent-core/src/features/metrics/instrument/index.js

```diff
@@ -92,6 +94,15 @@ export class Instrument extends FeatureBase {
         this.recordSupportability(`Generic/Polyfill/ObjectAssign/${this.PfFeatStatusEnum.OBJECT_ASSIGN}`);
         this.recordSupportability(`Generic/Polyfill/ObjectEntries/${this.PfFeatStatusEnum.OBJECT_ENTRIES}`);
     }
+
+    eachSessionChecks() {
+        // [Temporary] Report restores from BFCache to NR1 while feature flag is in place in lieu of sending pageshow events.
+        windowAddEventListener('pageshow', (evt) => {
```

Comment:
Metrics feature is loaded in workers, this might break that

Reply:
Gooood catch.